gradle/releaser.gradle

```diff
@@ -107,14 +107,36 @@ if (rootProject.hasProperty("artifactory_publish_password")) {
 			contextUrl = "${artifactory_publish_contextUrl}"
 			publish {
 				repository {
-					repoKey = "${artifactory_publish_repoKey}"
+					repoKey = repoKeyFor(p)
 					username = "${artifactory_publish_username}"
 					password = "${artifactory_publish_password}"
 				}
 			}
 			clientConfig.setIncludeEnvVars(false)
-			clientConfig.info.setBuildName('Reactor - Releaser - Addons')
+			clientConfig.info.setBuildName('Reactor - GitHub Actions - Addons')
 			clientConfig.info.setBuildNumber(buildNumber)
+			if (System.getenv("GITHUB_ACTIONS") == "true") {
+				clientConfig.info.setBuildUrl(System.getenv("GITHUB_SERVER_URL") + "/" + System.getenv("GITHUB_REPOSITORY") + "/actions/runs/" + System.getenv("GITHUB_RUN_ID"))
+				clientConfig.info.setVcsRevision(System.getenv("GITHUB_SHA"))
+			}
 		}
 	}
-}
\ No newline at end of file
+}
+
+static def repoKeyFor(p) {
+	def repos = [
+			// New scheme
+			(~/^\d+\.\d+\.\d+$/) : "libs-release-local",
```

Comment:
FYI Groovy supports patterns in switch's cases, so it can be changed to something like this:
```groovy
switch (p.version) {
    case (~/^\d+\.\d+\.\d+$/):
    case (~/^\d+\.\d+\.\d+\.RELEASE$/):
        return "libs-release-local";
    <...>
    default:
        throw new IllegalStateException("Unknown pattern");
}
```

it comes with an additional benefit of multiple cases returning the same value

Reply:
ah, we can do that as a follow up improvement